scripts/changelog.rb

```diff
@@ -9,6 +9,8 @@
 author_email = ENV['DEBEMAIL'] || ENV['EMAIL'] || ENV['GIT_AUTHOR_EMAIL'] || `git config --get user.email`.chomp
 options[:author] = "#{author_name} <#{author_email}>" if !author_name.nil? && !author_email.nil?
 
+VERSION_REGEX = %r{.*\((?<version>.*)-(?<release>.*)\).*}
```

Comment:
```suggestion
VERSION_REGEX = %r{\((?<version>.*)-(?<release>.*)\)}
```

the `.*` seem unnecessary here